iphone/Classes/TiUIView.m

```diff
@@ -1262,16 +1261,18 @@ - (void)touchesBegan:(NSSet *)touches withEvent:(UIEvent *)event
 }
 
 - (void)processTouchesBegan:(NSSet *)touches withEvent:(UIEvent *)event
-{
-    UITouch *touch = [touches anyObject];
-	
+{	 
 	if (handlesTouches)
 	{
-		NSDictionary *evt = [TiUtils pointToDictionary:[touch locationInView:self]];
+        UITouch *touch = [touches anyObject];
+        NSDictionary *evt = [TiUtils touchPropertiesToDictionary:touch:[touch locationInView:self]];
+
```

Comment:
Indentation level again, also: Why did the {{touch}} variable went from outside inside the handleTouches statement?